packages/gatsby/cache-dir/gatsby-browser-entry.js

```diff
@@ -57,7 +61,7 @@ const useStaticQuery = query => {
         `Please update React and ReactDOM to 16.8.0 or later to use the useStaticQuery hook.`
     )
   }
-  const context = React.useContext(StaticQueryContext)
+  const context = React.useContext(StaticQueryContext) || {}
```

Comment:
Why fallback is needed here?

Reply:
it probably doesn't — I think I added it when I was trying to debug something and it stayed. Removing.